spec/requests/api/graphql_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe ManageIQ::API::Common::GraphQL, :type => :request do
   let!(:graphql_endpoint) { "/api/v1.0/graphql" }
+  let!(:default_as)   { :text }
```

Comment:
Shouldn't this be json for now?

Reply:
somehow all graphql specs fail as follows without that:

```
      Failure/Error: ActionController::Parameters.new(parsed_body).permit!
      
      NoMethodError:
        undefined method `with_indifferent_access' for #<String:0x00007f865cf7a208>

```

ugly stack trace: 

```
      # /Users/abellotti/.gem/ruby/2.4.6/gems/actionpack-5.2.3/lib/action_controller/metal/strong_parameters.rb:236:in `initialize'
      # ./lib/manageiq/api/common/application_controller_mixins/request_body_validation.rb:33:in `new'
      # ./lib/manageiq/api/common/application_controller_mixins/request_body_validation.rb:33:in `body_params'
      # ./lib/manageiq/api/common/application_controller_mixins/request_body_validation.rb:51:in `validate_request'
      # /Users/abellotti/.gem/ruby/2.4.6/gems/activesupport-5.2.3/lib/active_support/callbacks.rb:426:in `block in make_lambda'
      # /Users/abellotti/.gem/ruby/2.4.6/gems/activesupport-5.2.3/lib/active_support/callbacks.rb:198:in `block (2 levels) in halting'
   ...
      # /Users/abellotti/.gem/ruby/2.4.6/gems/actionpack-5.2.3/lib/action_dispatch/testing/integration.rb:350:in `block (2 levels) in <module:Runner>'
      # ./spec/support/default_as_json.rb:13:in `block (2 levels) in <module:RequestHelpersCustomized>'
      # ./spec/requests/api/graphql_spec.rb:159:in `block (3 levels) in <top (required)>'

```

Reply:
fixed.  needed to remove the to_json for all the params in the graphql tests.